textgrad/engine/vllm.py

```diff
@@ -0,0 +1,50 @@
+try:
+    from vllm import LLM, SamplingParams
+    from transformers import AutoTokenizer
+except ImportError:
+    raise ImportError("If you'd like to use hugging face models, please install the vllm package by running `pip install vllm`")
+
+
+
+from .base import EngineLM, CachedEngine
+
+
+class ChatVllm(EngineLM, CachedEngine):
+    SYSTEM_PROMPT = "You are a helpful, creative, and smart assistant."
+
+    def __init__(
+        self,
+        model_string="microsoft/Phi-3-mini-4k-instruct",
+        system_prompt=SYSTEM_PROMPT,**kwargs
+    ):
+                
+        self.model_string = model_string
+        self.model = LLM(model_string,**kwargs)
```

Comment:
what kwargs do we expect here? Can we add some simple docs / instructions for this?